src/SourceDialog/SourceDialog.vala

```diff
@@ -24,7 +24,7 @@ public class Maya.View.SourceDialog : Gtk.Grid {
 
     private Gtk.Entry name_entry;
     private bool set_as_default = false;
-    private string hex_color = "#da3d41";
+    private string hex_color = "#ed5353";
```

Comment:
Do we need to adjust the color value to match the ColorButton.css here? I'm not quite sure.